Packages/MovieDBUI/Sources/MovieDBUI/Utils/Extentions/UIImageView+Extention.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIImageView+Extention.swift
+//  MovieDB
+//
+//  Created by Mouad Bj on 16/3/2023.
+//
+
+import UIKit
+
+extension UIImageView {
+  
+  func load(url: URL, placeholder: UIImage?, cache: URLCache? = nil) {
+    let cache = cache ?? URLCache.shared
+    let request = URLRequest(url: url)
+    
+    if let data = cache.cachedResponse(for: request)?.data, let image = UIImage(data: data) {
```

Comment:
Definitely, this is just a basic one for showcase, what is important here is to make the implementation in a single file (ideally), so it could be swappable if needed with little effort